README.md

```diff
@@ -0,0 +1,10 @@
+# COMS 3157 Guides
+
+This repository is a collection of guides for Advanced Programming (COMS 3157)
+developed by the TAs for Prof. Brian Borowski's offering of the course.
+
+
+## Programming Enviroment
+
+- [Login](login.md): Introduction on how to access the AP server.
+- [Git](B-Makefiles/makefiles.md): Makefiles.
```

Comment:
should Makefiles be a different item. Also, that's a broken link.

Reply:
whoops my b, i added a broken link checker to avoid this issues.